cg/services/decompression_service/decompressor.py

```diff
@@ -0,0 +1,20 @@
+import zipfile
+from pathlib import Path
+
+
+class Decompressor:
+
+    def decompress(self, source_path: Path, destination_path: Path) -> Path:
+        """Decompress a file to a directory."""
+        with zipfile.ZipFile(source_path, "r") as zip_file:
+            zip_file.extractall(destination_path)
+        if not self._decompressed_path_exists(destination_path):
```

Comment:
A bit uncertain of the timings of it all, but is the decompression async?

Reply:
Or could the decompression be running here still?

Reply:
As far as I understand it just unzips the folder, almost instant?